book-2-giffygram/chapters/47-GG-ERD.md

```diff
@@ -6,58 +6,66 @@ In this chapter, you are going to build a visualization of the data and how reso
 
 ## Learning Objectives
 You should be able to:
-* state the purpose of an ERD
-* define what is a resource for an application
-* remember that an ERD shows properties and relationships
+* State the purpose of an ERD
+* Define what is a "resource" for an application
+* Remember that an ERD shows properties and relationships
 
 ## Application Resources
 
 An Entity Relationship Diagram (ERD) is a tool for software developers to visualize the resources for an application and how they relate.
 
-What is a resource? It is the kind of data your users will view,create, delete, or update in your application.
+What is a resource? It is the kind of data your users will view, create, delete, or update in your application.
 
 For Giffygram, here is the list of the main resources that need to be represented in your code.
 
 1. Users - People who use the application.
-1. Posts - People will create new posts with animation GIFs in them.
-1. Messages - People will send messages to other users of the application.
+1. Posts - Users will create new Posts with animation GIFs in them.
+1. Messages - Users will send Messages to other Users of the application.
 
-Additionally, a major feature of this application is that users should have the ability to like posts from other users, just as you see on the major social media platforms.
+Additionally, a major feature of this application is that Users should have the ability to like Posts from other Users, just as you see on the major social media platforms.
 
 To visualize these resources and relationships, you are going to be using a diagramming tool called [DBDiagram](https://dbdiagram.io/).
 
-
 ## ERD Table Definitions
 
-Each resource in an application gets a table. For GiffyGram, we will start with tables for users, posts, messages.
+Each resource in an application gets a table. For Giffygram, we will start with tables for Users, Posts, and Messages.
 
-When building the table, consider what does one "user" or "post" look like. What are the properties? Also, each entry will have a unique id.
+When building the table, consider what one "User" or "Post" looks like. What are the properties? Also, each entry will have a unique identifier, or "id".
 
-![user and post visual](./images/47-Resource.png)
+![User and Post visual](./images/47-Resource.png)
 
-Open DBDiagram and create a table for users and posts. You can past the following definitions into the left panel.
+Open DBDiagram and create a table for Users and a table for Posts. You can paste the following definitions into the left panel.
 
 ```ddl
-Table users {
+Table Users {
```

Comment:
the resource or table name in the JSON file I've typically seen in lowercase.
not a big deal, but when we translate the ERD into JSON data, it may help to keep them consistent

Reply:
👍